keyboards/Emi20/kb.c

```diff
@@ -0,0 +1 @@
+#include "emi20.h"
```

Comment:
This file needs to be renamed.

Reply:
From kb.c to emi20.c I would imagine or do you mean the Emi20.h to emi20.h? Emi20.h has been renamed

Reply:
To `emi20.c`, yes.